bofire/surrogates/feature_importance.py

```diff
@@ -40,6 +40,13 @@ def permutation_importance(
         RegressionMetricsEnum.MSD: -1.0,
         RegressionMetricsEnum.PEARSON: 1.0,
         RegressionMetricsEnum.SPEARMAN: 1.0,
+        RegressionMetricsEnum.PEARSON_UQ: 1.0,
```

Comment:
Remove them here, it makes no sense for permutation importance. You are right, we need a second enum and second dictionary.

Reply:
We would then iterate here only over the standard regression metrices.